### DISCUSSION TITLE
Update Application - Windows

### DISCUSSION THREAD

**Discussion by azz22** [meisnate12/Plex-Meta-Manager#827] [General]
Hi, can i just copy and overwrite existing files to update plex meta manager or is there a script to run? my build is in windows with python. Appreciate your help in advance, thanks

**Comment by chazlarson** [comment]
Yes, you can do that.
If you installed via `git`, you could update with `git pull`
If you run with docker, `docker pull meisnate12/plex-meta-manager`
